comid-code-points.cddl

```diff
@@ -90,8 +90,7 @@ comid.instance-claims = 3
 comid.device-id = 1
 comid.key-material = 2
 
-; instance-value-map and $instance-value-choice
-comid.instance-element-name = 0
+; $instance-value-choice
```

Comment:
There is no such thing as $instance-value-choice, can we say ; member of $$instance-value-group-choice ??